Remotion/Core/Tools/Console/CommandLine/CommandLineArgumentCollection.cs

```diff
@@ -38,9 +38,9 @@ public CommandLineArgumentCollection ()
   {
   }
 
-  public CommandLineArgument this[int index]
+  public CommandLineArgument? this[int index]
```

Comment:
In what instance may this be null?

Reply:
`IList.this[]` returns `object?`, that's why I inferred it here, but you're right, `null` values are quite easily controllable if the collection is used right.


Changed to the following:
```csharp
public CommandLineArgument this[int index]
{
  get { return (CommandLineArgument) List[index]!; }
  set { List[index] = value; }
}
```